bin/n

```diff
@@ -16,8 +16,8 @@ BASE_VERSIONS_DIR=$N_PREFIX/n/versions
 
 BINS=("node"
       "io")
-MIRROR=(${NODE_MIRROR-https://nodejs.org/dist/}
-        ${IO_MIRROR-https://iojs.org/dist/})
+MIRROR=(${NODE_MIRROR-http://npm.taobao.org/mirrors/node/}
+        ${IO_MIRROR-http://npm.taobao.org/mirrors/iojs/})
```

Comment:
I think you added this to this PR by accident, you can avoid these situations by always creating a new branch in your fork for each feature you want to create a PR for.